types/enzyme-adapter-react-16/index.d.ts

```diff
@@ -1,15 +1,15 @@
-// Type definitions for enzyme-adapter-react-16 1.0
+// Type definitions for enzyme-adapter-react-16 1.6
 // Project: https://github.com/airbnb/enzyme
 // Definitions by: Tanguy Krotoff <https://github.com/tkrotoff>
+//                 Jimmy Headdon <https://github.com/jimmyheaddon>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import { EnzymeAdapter } from 'enzyme';
 
 declare class ReactSixteenAdapter extends EnzymeAdapter {
+    constructor();
```

Comment:
Don't think you need this. Please remove it. (EnzymeAdapter has same signature of constructor and ReactSixteenAdapter will inherit it

Reply:
Thanks both, I'll remove that now!